lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1372,13 +1372,15 @@ paths:
         - EnodeBs
       parameters:
         - $ref: './orc8r-swagger-common.yml#/parameters/network_id'
+        - $ref: './orc8r-swagger-common.yml#/parameters/page_size'
+        - $ref: './orc8r-swagger-common.yml#/parameters/page_token'
       responses:
         '200':
-          description: All enodeBs registered in the network
+          description: All enodeBs registered in the network with pagination support
```

Comment:
It doesn't actually return "all" enodebs in each response. So I would say "List eNodeBs registered..."